crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -907,6 +919,33 @@ impl WindowBuilder for WindowBuilderWrapper {
     self
   }
 
+  /// Prevent the window from overflowing the working area (e.g. monitor size - taskbar size) on creation
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux:** Prevent overflowing the monitor instead of workarea
+  /// - **iOS / Android:** Unsupported.
+  #[must_use]
+  fn prevent_overflow(mut self) -> Self {
+    self
+      .prevent_overflow
+      .replace(PhysicalSize::new(0, 0).into());
+    self
+  }
+
+  /// Prevent the window from overflowing the working area (e.g. monitor size - taskbar size)
+  /// on creation with a margin
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux:** Prevent overflowing the monitor instead of workarea
```

Comment:
why? can't we use https://docs.rs/gdk/0.18.0/gdk/prelude/trait.MonitorExt.html#method.workarea

Reply:
Ah, didn't know it exits, do you think we can move this `get_work_area_size` to `tao` instead? If not we'll need a way to access the inner `gdk::Monitor` from `tao`

Reply:
we can expose the raw `gdk::Monitor` in `tao` similar to `MonitorHandleExtWindows::hmonitor`